spec/comm.mediawiki

```diff
@@ -187,11 +187,24 @@ the client's absence. A list of any pending matches is included in the response.
 {|
 ! field   !! type   !! description
 |-
-| matches || &#91;object&#93; || list of [[orders.mediawiki/#Match_negotiation|Match objects]]
+| matches || &#91;object&#93; || list of [[orders.mediawiki/#Match_Negotiation|Match objects]]
 |-
 | sig     || string || hex-encoded server's signature of the serialized connection data
 |}
 
+==Acknowledgements==
+
+Often, the server only requires a simple signed acknowledgement for a request.
+The specification will provide instructions for serialization of the request
+data. The client will then sign the serialized data with their private key, and
+send an acknowledgement of the following form.
```

Comment:
In the case of the preimage ack, the client and server are reversed.  If we do the preimage ack, how about making this paragraph generic, in terms of peers.